src/code/stores/graph-store.ts

```diff
@@ -666,6 +669,16 @@ export const GraphStore: GraphStoreClass = Reflux.createStore({
     return this.undoRedoManager.endCommandBatch();
   },
 
+  startNodeSliderDrag(node: Node) {
+    _.map(node.outNodes(), (outNode: Node) => outNode.unscaled = true);
```

Comment:
Since you're not using the returned array, `_.each()` seems more appropriate here and in `endNodeSliderDrag()` below.

Reply:
Thanks.  @kswenson - I just pushed this up to make it available during a meeting looking into how the models work - its not anywhere near done.